uiowa/uiowa
# [Facilities] Artwork list view fix

Resolves https://github.com/uiowa/uiowa/issues/7260. 

# How to test

```
ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local uli /art-campus-gallery
```
- Code review
- Compare against https://facilities.prod.drupal.uiowa.edu/art-campus-gallery and confirm that only person types with "artist" person type are shown as options.
